stdlibs/tests/unistd.h

```diff
@@ -0,0 +1,9 @@
+
+#define F_OK 0
+#define R_OK 1
+#define W_OK 2
+#define X_OK 3
+
+int access(char * path, int amode) {
+  return 0;
```

Comment:
test `path` is `/some/path` and `amode` is `F-OK`

Reply:
Validated